### DISCUSSION TITLE
Unable to install wine in Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by Koinonia-1** [PR3SIDENT/enshrouded-server#36] [Q&A]
I tried installing a dedicated server on Ubuntu 20.04, but I got stuck in the wine installation part.

See log below

**root@instance-1:/etc/apt# apt install -y --install-recommends winehq-staging
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 winehq-staging : Depends: wine-staging (= 9.0.0~jammy-1)
E: Unable to correct problems, you have held broken packages.**

**Comment by lipe123** [comment]
So your install is bad, sounds like you need to force remove all the wine packages and dependencies and try again.
I did a clean server install of ubuntu 22.04 and it worked first time with no issues at all.

**Comment by kelposun** [comment]
> I tried installing a dedicated server on Ubuntu 20.04, but I got stuck in the wine installation part.
> 
> See log below
> 
> **root@instance-1:/etc/apt# apt install -y --install-recommends winehq-staging Reading package lists... Done Building dependency tree Reading state information... Done Some packages could not be installed. This may mean that you have requested an impossible situation or if you are using the unstable distribution that some required packages have not yet been created or been moved out of Incoming. The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies: winehq-staging : Depends: wine-staging (= 9.0.0~jammy-1) E: Unable to correct problems, you have held broken packages.**

Just change
`wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources`
to
`wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/focal/winehq-focal.sources`

**Reply by azazely85** [reply]
did  not work same problem
